packages/amplify_analytics_pinpoint/.packages

```diff
@@ -0,0 +1,50 @@
+# This file is deprecated. Tools should instead consume 
```

Comment:
Pretty sure this file was accidentally generated and committed. `amplify_analytics_pinpoint` is being moved to the `amplify_analytics_pinpoint_plugin` directory.